### PR TITLE
Fix #1810: Renaming the admin profile creates new one

### LINKED ISSUE #1810: Renaming the admin profile creates a new one
**Describe the bug**
Renaming the admin profile creates a new one

**To Reproduce**
1. Log into the account account
2. Open nav drawer
3. Go to admin controls
4. Go to edit profiles
5. Select admin profile
6. Rename the profile
7. Navigate back to the profile chooser
8. Observe there are now two admin accounts (note that both of these have administrator privileges, too)

**Expected behavior**
Renaming shouldn't duplicate accounts, especially the administrator (it seems to work correctly for regular profiles).

**Device**
- Device: Pixel XL
- SDK version: 29

### PR DESCRIPTION
**Pull request by chrk2205**
## Explanation
fixes #1810 
to overcome this, I created a function for `addProfile` to check if an admin.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@chrk2205 Please have a look at lint tests which are failing. Thanks.

**Comment by rt4914** [comment]
@chrk2205 
1. Please have a look at lint tests which are failing. 
2. I think we should add tests for this functionality in `ProfileManagementControllerTest`.
3. Also, the PR title should be :
`Fix #1810: Renaming the admin profile creates new one`

**Comment by chrk2205** [comment]
> @chrk2205 Please have a look at lint tests which are failing. Thanks.

**Exceeded max line length**  in
`return@storeDataWithCustomChannelAsync Pair(it, ProfileActionStatus.PROFILE_ALREADY_HAS_ADMIN)`

**Comment by chrk2205** [comment]
> @chrk2205 
> 2. I think we should add tests for this functionality in `ProfileManagementControllerTest`.

okay.

**Comment by chrk2205** [comment]
@rt4914 @BenHenning 
testcase Added &
all lint cases passed
PTAL thankyou

File: domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt [hunk 1]
```diff
@@ -729,6 +729,33 @@ class ProfileManagementControllerTest {
     assertThat(wasProfileEverAdded).isTrue()
   }
 
+  @Test
+  fun testAddAdminProfile_addAnotherAdminProfile_checkSecondAdminProfileWasAdded() {
```
Comment (rt4914) [inline domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt]: Rename this to `testAddAdminProfile_addAnotherAdminProfile_checkSecondAdminProfileWasNotAdded `

You missed **Not** in the sentence.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Really nice work. One nit suggested.

File: domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt (hunk 1 above)
Comment (chrk2205) [reply domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt]: > Really nice work. One nit suggested.

Thankyou @rt4914 
nit changes done!!

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**